assets/css/base/_base.scss

```diff
@@ -1512,14 +1512,11 @@ button.menu-toggle {
 	}
 }
 
-.widget-area,
-.footer-widgets {
-	#wp-calendar {
-		th,
-		td {
-			padding: 0.236em;
-			text-align: center;
-		}
+.wp-calendar-table {
```

Comment:
This breaks <5.4, since the `.wp-calendar-table` classes got added in 5.4. I downgraded WP and this is what the calendar block looks like (on front end):

<img width="552" alt="Screen Shot 2020-04-20 at 11 57 22 AM" src="https://user-images.githubusercontent.com/4167300/79703191-49d19f00-82fe-11ea-9410-f9400c024b5b.png">

Perhaps we need to set up our shared calendar styling for `.wp-calendar-block, .wp-calendar-table` so it applies to both (even in older WordPress).

Up to you how to handle this – if this is complex and we have slightly different/degraded styles in older WordPress that's probably ok to ship.

Reply:
Absolutely, I should have thought about that. I added back a selector targeting WP <=5.3 calendar widget (1b6e26f7b2370f8f605a64c4d3f2e9d0dc473016). I added a comment specifying that selector is only needed for WP <=5.3 in case at some point we drop support for old versions and we can remove it.